tests/test_eof.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from dahuffman.huffmancodec import _EOF
+
+
+def test_eq():
```

Comment:
I don't think it's necessary to put these tests in a separate file.
I would just add them in the existing test_dahuffman.py file
Maybe move the tests and fixtures inside a class, e.g. `TestEndOfFileSymbol`. I'm guessing you used a separate file to keep the fixtures away from existing tests, but you can also achieve that by scoping them to a single class